imgix.go

```diff
@@ -153,11 +154,22 @@ func (c *Client) PathWithParams(imgPath string, params url.Values) string {
 
 	urlString += imgPath
 
+	for key, val := range params {
+		if strings.HasSuffix(key, "64") {
+			valData := []byte(val[0])
+			base64EncodedVal := base64.URLEncoding.EncodeToString(valData)
+			base64EncodedVal = strings.Replace(base64EncodedVal, "=", "", -1)
+			params.Set(key, base64EncodedVal)
+		}
+	}
+
 	// The signature in an imgix URL must always be the **last** parameter in a URL,
 	// hence some of the gross string concatenation here. net/url will aggressively
 	// alphabetize the URL parameters.
 	signature := c.SignatureForPathAndParams(imgPath, params)
 	parameterString := params.Encode()
+	parameterString = strings.Replace(parameterString, "+", "%%20", -1)
```

Comment:
I may be misunderstanding Go string escaping, but it seemed that the `%%` was required in order to properly output the final `%20` string in the output. Is this not the case? The tests show that this works, but again, I might totally be missing something.

As for the rationale, either are technically valid. In our particular case, encoding space characters as `%20` helps keeps output consistent among all our libraries.